### PR TITLE
Add Feeder IAM task role

### PR DESCRIPTION
**Pull request by farski**

### CODE REVIEW

File: stacks/feeder.prx.org.yml [hunk 1]
```diff
@@ -330,37 +405,21 @@ Resources:
...
         - Key: Environment
           Value: !Ref EnvironmentType
-        - Key: "prx:cloudformation:stack-name"
+        - Key: prx:cloudformation:stack-name
           Value: !Ref AWS::StackName
-        - Key: "prx:cloudformation:stack-id"
+        - Key: prx:cloudformation:stack-id
           Value: !Ref AWS::StackId
       TaskDefinition: !Ref WebTaskDefinition
-  # Route 53
```
Comment (farski) [inline stacks/feeder.prx.org.yml:338]: This isn't gone, just moved down the file

File: stacks/feeder.prx.org.yml [hunk 2]
```diff
@@ -264,13 +263,85 @@ Resources:
...
+                Resource:
+                  - "*_announce_feed_*"
+        - PolicyName: S3FeedBucketAccess
+          PolicyDocument:
+            Version: "2012-10-17"
+            Statement:
+              - Effect: Allow
+                Action:
+                  - s3:AbortMultipartUpload
+                  # - s3:CreateBucket
```
Comment (farski) [inline stacks/feeder.prx.org.yml:305]: I commented out permissions that the old key has but didn't seem necessary

File: stacks/feeder.prx.org.yml [hunk 3]
```diff
@@ -264,13 +263,85 @@ Resources:
       Dimensions:
         - Name: LoadBalancer
           Value: !GetAtt ALB.LoadBalancerFullName
+
```
Comment (cavis) [inline stacks/feeder.prx.org.yml:266]: The linter allows empty space?  Bwah?

**Review by cavis — APPROVED** [review]
👍 Looks good to me!

**Pull request merged**